console-csharp-connect-sample/MailHelper.cs

```diff
@@ -111,11 +115,9 @@ public static Stream GetCurrentUserPhotoStreamAsync()
 
             try
             {
-                var graphClient = AuthenticationHelper.GetAuthenticatedClient();
-                currentUserPhotoStream = graphClient.Me.Photo.Content.Request().GetAsync().Result;
+				currentUserPhotoStream = _graphServiceClient.Me.Photo.Content.Request().GetAsync().Result;
```

Comment:
you can drop the .Result and add an await keyword.

Reply:
This is done